marconi-chain-index/src/Marconi/ChainIndex/Experimental/Runner.hs

```diff
@@ -87,9 +87,8 @@ runIndexer trace securityParam retryConfig socketPath networkId startingPoints i
             PP.renderStrict $
               PP.layoutPretty PP.defaultLayoutOptions $
                 PP.pretty NoIntersectionFoundLog
-    Async.concurrently_
-      (void $ runChainSyncStream `catch` whenNoIntersectionFound)
-      (Core.processQueue eventQueue cBox)
+    void $ Concurrent.forkIO $ runChainSyncStream `catch` whenNoIntersectionFound
+    Core.processQueue eventQueue cBox
```

Comment:
Why change this?

Reply:
with `Async` SIGTERM are sometime caught and hidden to the main thread, and we need to send a second one.

Reply:
:+1: